src/components/ChatMessages.tsx

```diff
@@ -14,6 +15,28 @@ interface IChatMessagesProps {
   messages: IMessage[];
 }
 
+// Code component for ReactMarkdown
+const MarkdownCode: React.FC<{
+  inline?: boolean;
+  className?: string;
+  children?: React.ReactNode;
+}> = ({ inline, children }) => {
+  if (inline) {
+    return <code className="rounded bg-gray-200 px-1 py-0.5">{children}</code>;
+  }
+
+  return (
+    <code className="block overflow-x-auto rounded bg-gray-900 p-2 text-white">
+      {children}
+    </code>
+  );
+};
+
+// Components object for ReactMarkdown
+const markdownComponents: Components = {
+  code: MarkdownCode
+};
+
```

Comment:
i think the markdown-related functionality should get refactored to its own file (e.g. `Markdown.tsx`), and in this file we just render it, something like:

```tsx
<Markdown text={message.text} />
```